src/main/java/cz/muni/fi/pa165/dao/PerformanceDao.java

```diff
@@ -0,0 +1,23 @@
+package cz.muni.fi.pa165.dao;
+
+import java.util.List;
+
+import cz.muni.fi.pa165.entity.Hall;

```

Comment:
Pokud je toho vic ze stejneho package, tak je lepsi importovat * misto trid. To se da ale upravit pak jednotne, ze to zformatujeme na cisto, az to bude hotove vsechno

Reply:
Snad na první javě, tak nám myslím vštěpovali, ať se snažíme importovat co možná nejvíce bez *. Samozřejmě pokud importuješ víc jak 3 classy z jednoho balíku, tak už tam * dát můžeš. 
Mě je to celkem jedno, klidně to pak všude zformátujeme.